pyxem/generators/indexation_generator.py

```diff
@@ -95,29 +100,29 @@ def correlate(self,
             # Index at all real space pixels
             mask = 1
 
-        # TODO: Add extra methods
-        no_extra_methods_yet = True
-        if no_extra_methods_yet:
-            # adds a normalisation to library
-            for phase in library.keys():
-                norm_array = np.ones(library[phase]['intensities'].shape[0])  # will store the norms
-                for i, intensity_array in enumerate(library[phase]['intensities']):
-                    norm_array[i] = np.linalg.norm(intensity_array)
-                library[phase]['pattern_norms'] = norm_array  # puts this normalisation into the library
-
-            matches = signal.map(correlate_library,
-                                 library=library,
-                                 n_largest=n_largest,
-                                 mask=mask,
-                                 inplace=False,
-                                 **kwargs)
+        # adds a normalisation to library
+        for phase in library.keys():
+            norm_array = np.ones(library[phase]['intensities'].shape[0])  # will store the norms
+
+            for i, intensity_array in enumerate(library[phase]['intensities']):
+                norm_array[i] = np.linalg.norm(intensity_array)
+            library[phase]['pattern_norms'] = norm_array  # puts this normalisation into the library
+
+        matches = signal.map(correlate_library,
+                             library=library,
+                             n_largest=n_largest,
+                             mask=mask,
+                             inplace=False,
+                             method = method,
```

Comment:
can we have `method` above `inplace` please. Another boring convention that makes the code easier to read. The arguments that are for `map` come after those for functions